test/xref2/module_list.t/main.mli

```diff
@@ -56,3 +57,10 @@ module Inline_include : sig
   include T
   (** @inline *)
 end
+
+module Resolve_synopsis : sig
+  (** This should be resolved when included: {!Main.Resolve_synopsis.t}. These
+      shouldn't: {!t} {!Resolve_synopsis.t} *)
```

Comment:
It should be easy to resolve these two references:
- Add the target module to the new env
- Open it

How sound it is ?

Reply:
Let's leave this as an issue to resolve later - we need to discuss the various options available to us.